content/security/vulnerabilities.adoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: security
+title: "Vulnerabilities and Scoring"
+section: security
+---
+
+This page collects information about historically common kinds of security vulnerabilities in Jenkins, including how we score them.
+For vulnerability scoring, we use https://www.first.org/cvss/v3-1/[CVSS v3.1].
+
+[#xss]
+== Cross-Site Scripting (XSS)
+
+Attackers can inject JavaScript into Jenkins in a way that causes it to be executed in another user's (the "victim") browser when they view certain pages.
+https://developer.mozilla.org/en-US/docs/Web/Security/Attacks/XSS[Learn more].
+
+Who can exploit this vulnerability depends on whether it is _stored_ or _reflected_.
+The latter can generally be exploited by anyone, usually by having the victim click a link (CVSS v3.1: `PR:N/UI:R`), while the former usually needs some permission, e.g., View/Configure, to store the payload (CVSS v3.1: `PR:L/UI:R` or `PR:H/UI:R`).
```

Comment:
```suggestion
The latter can generally be exploited by anyone, usually by having the victim click a link (CVSS v3.1: `PR:N/UI:R`), while the former usually needs some permission, such as View/Configure, to store the payload (CVSS v3.1: `PR:L/UI:R` or `PR:H/UI:R`).
```
Suggestion to replace `e.g.` with `such as` per [microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)